### PR TITLE
PHPC-655: Use case insensitive parsing for Manager connectTimeoutMS array option

### PR DESCRIPTION
**Pull request by jmikola**
https://jira.mongodb.org/browse/PHPC-655

### CODE REVIEW

File: tests/standalone/bug0655.phpt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+<?php if (defined("HHVM_VERSION_ID")) exit("skip HHVM does not use custom streams"); ?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$command = new MongoDB\Driver\Command(['ping' => 1]);
+
+// Invalid host cannot be resolved
+$manager = new MongoDB\Driver\Manager('mongodb://invalid.host:27017', ['connectTimeoutMS' => 1]);
```
Comment (derickr) [inline tests/standalone/bug0655.phpt]: Shouldn't you test multiple variants of the casing?

**Comment by derickr** [comment]
LGTM, with perhaps adding tests for different cases of ``connectTimeoutMS``.

File: tests/standalone/bug0655.phpt (hunk 1 above)
Comment (jmikola) [inline tests/standalone/bug0655.phpt]: I didn't see the point, as the code (our own and `mongoc-uri.c`) uses all-lowercase string literals. I can add a few, though.

Comment (jmikola) [inline tests/standalone/bug0655.phpt]: I modified the test below to use "CONNECTTIMEOUTMS".

**Comment by jmikola** [comment]
Manually merged to `v1.1` in https://github.com/mongodb/mongo-php-driver/commit/82d38745073a59af7b233242b53ca6a8cb91c9c6 and merged up to master in https://github.com/mongodb/mongo-php-driver/commit/4a64b3d300607b01db28a0dc753939aa37ab2399.

**Pull request closed without merging**